packages/sommelier.rb

```diff
@@ -230,14 +221,14 @@ def self.install
       system "DESTDIR=#{CREW_DEST_DIR} ninja -C builddir install"
       Dir.chdir('builddir') do
         FileUtils.mv "#{CREW_DEST_PREFIX}/bin/sommelier", "#{CREW_DEST_PREFIX}/bin/sommelier.elf"
-        system "install -Dm755 sommelier_sh #{CREW_DEST_PREFIX}/bin/sommelier"
-        system "install -Dm755 sommelierd #{CREW_DEST_PREFIX}/sbin/sommelierd"
-        system "install -Dm755 startsommelier #{CREW_DEST_PREFIX}/bin/startsommelier"
+        FileUtils.install 'sommelier_sh', "#{CREW_DEST_PREFIX}/bin/sommelier", mode: 0o755
+        FileUtils.install 'sommelierd', "#{CREW_DEST_PREFIX}/sbin/sommelierd", mode: 0o755
+        FileUtils.install 'startsommelier', "#{CREW_DEST_PREFIX}/bin/startsommelier", mode: 0o755
         FileUtils.ln_sf 'startsommelier', "#{CREW_DEST_PREFIX}/bin/initsommelier"
-        system "install -Dm755 stopsommelier #{CREW_DEST_PREFIX}/bin/stopsommelier"
-        system "install -Dm755 restartsommelier #{CREW_DEST_PREFIX}/bin/restartsommelier"
-        system "install -Dm755 sommelierrc #{CREW_DEST_PREFIX}/etc/sommelierrc"
-        system "install -Dm644 .sommelier-default.env #{CREW_DEST_HOME}/.sommelier-default.env"
+        FileUtils.install 'stopsommelier', "#{CREW_DEST_PREFIX}/bin/stopsommelier", mode: 0o755
+        FileUtils.install 'restartsommelier', "#{CREW_DEST_PREFIX}/bin/restartsommelier", mode: 0o755
+        FileUtils.install 'sommelierrc', "#{CREW_DEST_PREFIX}/etc/sommelierrc", mode: 0o755
+        FileUtils.install '.sommelier-default.env', "#{CREW_DEST_HOME}/.sommelier-default.env", mode: 0o644
```

Comment:
If we use FileUtils.install, don't we have to create directories first?  Does this compile without errors?